bats/fb-test-puppet.bats

```diff
@@ -105,3 +105,9 @@ fi
   puppet agent -v -o --no-daemonize
   grep -i "property of the Foreman project" /etc/motd
 }
+
+@test "ensure there is a report for Puppet" {
+  RESULT=$(hammer --output csv --no-headers config-report list --fields "Id" --search "host=${HOSTNAME} origin=Puppet")
```

Comment:
depending on the OS settings, `HOSTNAME` can be short or long, so to enforce the long notation, we have the following in the Ansible test `setup()` function:
https://github.com/theforeman/forklift/blob/89f3ab574d6a5551c9b724a06c3b707d739ec102/bats/fb-test-foreman-ansible.bats#L9-L10

Reply:
Updated